hono/test/utils/db.ts

```diff
@@ -0,0 +1,136 @@
+import { execSync } from 'child_process'
+import fs from 'fs'
+import { platform } from 'os'
+import { sql } from 'drizzle-orm'
+import { drizzle } from 'drizzle-orm/postgres-js'
+import { migrate } from 'drizzle-orm/postgres-js/migrator'
+import postgres from 'postgres'
+
+const TEMP_ROOT = `tmp-postgres-local`
+
+export async function start(
+  port: number = 5555,
+  version: number = 16,
+  includeInstallation: boolean = false
+): Promise<void> {
+  try {
+    execSync(getInstallationScript(port, version, includeInstallation))
+  } catch (e) {
+    stop(port, version)
+    throw e
+  }
+}
+
+export async function provision(name: string, port: number = 5555, version: number = 16) {
```

Comment:
I am fine with directly declaring variables to `5555` and `16`  but assigning them as constants at the top of the file might be better because those values are used throughout it.

Reply:
Good idea, added constants.